paper/paper.tex

```diff
@@ -591,32 +591,64 @@ \section{Numerical example}
-    \label{tab:benchmark}
+    \caption{\textcolor{magenta}{Comparaison between SCOTS, CoSyMA and \texttt{Dionysos.jl} (\texttt{AB.UniformGridAbstraction} solver from Table~\ref{tab:solvers}) for the path planning example. \texttt{Dionysos.jl} outperforms SCOTS.}}
+    \label{tab:vehicle_bench}
 \end{table}
 
-We see that \texttt{Dionysos.jl} outperforms SCOTS for these two examples. We also reproduced \cite[Figures 3 and 4]{rungger2016scots} in Figure~\ref{fig:dcdc} and Figure~\ref{fig:vehicle}, which shows that they compute the same controller. The reason for such a difference between SCOTS, written in C++, and \texttt{Dionysos.jl} does not lie in the programming language used to write the package but in the synthesis algorithm itself. For example, unlike SCOTS, our package does not make use of \emph{Binary Decision Diagrams} (or \emph{BDDs})~\cite{bryant1992symbolic}, which as recognized in~\cite{rungger2016scots} results in substantially longer execution times compared to tools that use alternative data structures. \textcolor{magenta}{Moreover, we see that the synthesis performance of \texttt{Dionysos.jl} is close to the one of CoSyMA on the DC-DC converter example, while the latter constructs approximate bisimilar abstractions, unlike \texttt{Dionysos.jl}.}
+\color{black}
+
+% \begin{table}[ht!]
+%     \centering
+%     \color{magenta}
+%     \begin{tabular}{cc|cc}
+%         & & Abstraction [s] & Synthesis [s] \\ 
+%         \hline 
+%         Path planning & \texttt{Dionysos.jl} & \textbf{26.58} & \textbf{31.46} \\
+%         & SCOTS & 116.55 & 522.09 \\
+%         \hline
+%         DC-DC & \texttt{Dionysos.jl} & \textbf{6.62} & \textbf{6.10} \\
+%         & SCOTS & 20.97 & 74.18 \\ 
+%         & CoSyMA & - & (5.97)
+%     \end{tabular}
+%     \color{black}
+%     \caption{Comparaison between SCOTS and \texttt{Dionysos.jl} (\texttt{AB.UniformGridAbstraction} solver from Table~\ref{tab:solvers}) for the two problems in \cite{rungger2016scots}. \textcolor{magenta}{CoSyMA is also executed on the DC-DC converter problem for information.}}
+%     \label{tab:benchmark}
+% \end{table}
+
+We see that \texttt{Dionysos.jl} outperforms the other packages for these two examples. We also reproduced \cite[Figures 3 and 4]{rungger2016scots} in Figure~\ref{fig:dcdc} and Figure~\ref{fig:vehicle}, which shows that they compute the same controller. The reason for such a difference between SCOTS, written in C++, and \texttt{Dionysos.jl} does not lie in the programming language used to write the package but in the synthesis algorithm itself. For example, unlike SCOTS, our package does not make use of \emph{Binary Decision Diagrams} (or \emph{BDDs})~\cite{bryant1992symbolic}, which as recognized in~\cite{rungger2016scots} results in substantially longer execution times compared to tools that use alternative data structures. 
```

Comment:
Add that the plot with and without prior for DC-DC are identical, as can be verified in \cite{code-ocean capsule} so only include one of the two in \cref{fig:dcdc}